markdown/events/2020/aws-webinar.md

```diff
@@ -16,7 +16,7 @@ location_url: https://pages.awscloud.com/Nordics_InSpotlight_WebinarSeries.html
 > the cost-effective, scalable, and secure compute, storage, and database capabilities needed to accelerate time-to-science.
 
 Phil Ewels ([@ewels](http://github.com/ewels/)) will give a talk about how the cloud is used at the
-[National Genomics Infrastructure](https://ngisweden.se), part of [SciLifeLab](https://www.scilifelab.se) in Sweden.
+[National Genomics Infrastructure](https://ngisweden.scilifelab.se/), part of [SciLifeLab](https://www.scilifelab.se) in Sweden.
```

Comment:
Oops, good to know! https://ngisweden.se should work, not sure why it's down at the moment.. I'll look into it.